src/gui/EffectView.cpp

```diff
@@ -255,19 +256,15 @@ void EffectView::mouseMoveEvent(QMouseEvent* event)
 
 void EffectView::paintEvent( QPaintEvent * )
 {
-	QPainter p( this );
-	p.drawPixmap( 0, 0, m_bg );
-
-	QFont f = adjustedToPixelSize(font(), 10);
-	f.setBold( true );
-	p.setFont( f );
+	QPainter p(this);
+	QPainterPath path;
 
-	QString elidedText = p.fontMetrics().elidedText( model()->displayName(), Qt::ElideRight, width() - 22 );
+	path.addRoundedRect(QRectF(2, 2, EffectView::DEFAULT_WIDTH - 4, EffectView::DEFAULT_HEIGHT - 4), 2, 2);
```

Comment:
Is it intentional that the upper left margin is 2 and the lower right 4?

By the way, you can write this code independent of `DEFAULT_WIDTH` and `DEFAULT_HEIGHT` by simply removing a certain margin from the full rectangle of the widget:
```
const int inset = 2;
path.addRoundedRect(rect().marginsRemoved(QMargins(inset, inset, inset, inset)), 2, 2);
```

Reply:
the margins aren't 2 and 4, they're 2 and 2 -- the first two coords are the top left corner, and the next two are length and width, not the bottom right corner. So in order to be 2 px away from the edges at the end, we need to subtract 4px (since we're already inset 2px from the top left.)

That being said the `marginsRemoved()` method is probably better.